qubes-rpc/qvm-open-in-dvm.desktop

```diff
@@ -0,0 +1,10 @@
+### Note: With this installed, typing "xdg-settings set default-web-browser qvm-open-in-dvm.desktop" will make it so that in gnome-terminal
+###   that right clicking on a URL on screen, then selecting "open with" to open the link, will open it in a disposable VM (as opposed to opening it in the current VM)
+###   (typing "xdg-settings set default-web-browser firefox.desktop" will put it back to normal)
+
+[Desktop Entry]
+Name=qvm-open-in-dvm
+Exec=qvm-open-in-dvm %u
+Type=Application
+Categories=Network;WebBrowser;
```

Comment:
is it possible not to specify the "browser category"? I can image in the future this default `.desktop` file being more broadly used.

Reply:
Perhaps it doesn't make much difference since AppVMs don't really have a menu which can sort by category. But I thought I'd mention it anyways.

Reply:
well this is to solve this issue:
#https://github.com/QubesOS/qubes-issues/issues/6586
with opening URLS.

Can you specify as to which ".desktop" file you are referring to as default (this .desktop file is *not* set as default for anything.  You must manually set it).  Also can you elaborate the "more broadly used" part as in more broadly used being a good thing or a bad thing?

Reply:
oh, is "Categories" a qubes thing?  I assumed it was a desktop/gnome/kde thing

Reply:
Not sure

Reply:
I got the .desktop file from https://github.com/SvenSemmler .  He had posted the file as part of a resolution for issue https://github.com/QubesOS/qubes-issues/issues/6586 .  (I realized it was of general use for many people, not just that issue, and am submitting it for inclusion)

Perhaps he knows?

Reply:
@SvenSemmler

Reply:
I am super confused, why is this in 'qubes-rpc'?

Reply:
qubes-rpc?  I'm not familiar with that, but I'm new so my knowledge of the codebase is limited.  I put it in qubes-core-agent-linux cause it's essentially just a wrapper for "qvm-open-in-dvm", and that's where qvm-open-in-dvm is located.  Is there a different repo that is automatically installed inside the VMs that I should have put it in?

Reply:
@ddevz @deeplow .. I am sorry but I don't think I can be of help in this context. My recommendation would be to send an email to qubes-devel describing what your intention is and asking for guidance where to locate the PR. @andrewdavidwong please advise if there is a better course of action.

Reply:
I found a repository:  
https://github.com/QubesOS/qubes-gui-agent-linux

It has no README.md but inferring from the name, maybe this is the correct place to put it?  If so, I can move it there.

Reply:
I'm sorry.  I just realized that you were asking why i was putting it in that directory.  I thought you were asking why I was putting it in that repository (out of the 151 to choose from).

You are correct.  I should have put it in /applications-dropins/
I'll change it